draft-ietf-quic-http.md

```diff
@@ -363,23 +498,25 @@ pushes via the SETTINGS_DISABLE_PUSH setting in the SETTINGS frame (see
 {{connection-establishment}}), which defaults to 1 (true).
 
 As with server push for HTTP/2, the server initiates a server push by sending a
-PUSH_PROMISE frame containing the Stream ID of the stream to be pushed, as well
-as request header fields attributed to the request. The PUSH_PROMISE frame is
-sent on the control stream of the associated (client-initiated) request, while
-the Promised Stream ID field specifies the Stream ID of the control stream for
-the server-initiated request.
+PUSH_PROMISE frame that includes request header fields attributed to the
+request. The PUSH_PROMISE frame is sent on a response stream.  Unlike HTTP/2,
+the PUSH_PROMISE does not reference a stream; when a server fulfills a promise,
+the stream that carries the stream headers references the PUSH_PROMISE.  This
+allows a server to fulfill promises in the order that best suits its needs.
 
-The server push response is conveyed in the same way as a non-server-push
-response, with response headers and (if present) trailers carried by HEADERS
-frames sent on the control stream, and response body (if any) sent via the
-corresponding data stream.
+The server push response is conveyed on a push stream.  Aside from the stream
+header, which identifies the response stream and PUSH_PROMISE frame, a push
+stream is identical to a regular response stream ({{stream-response}}), with
+response headers and (if present) trailers carried by HEADERS frames sent on the
+control stream, and response body (if any) indicated with a HAS_BODY frame and
+associated data stream.
 
 
 # HTTP Framing Layer
```

Comment:
I think the {{http-framing-layer}} link is intended to point here, but it doesn't have an anchor.